lib/bot.js

```diff
@@ -37,6 +41,14 @@ class Bot {
     this.ircStatusNotices = options.ircStatusNotices;
     this.announceSelfJoin = options.announceSelfJoin;
 
+    // IRC nicks to ignore (case insensitive)
+    this.ircIgnoreUsers = options.ircIgnoreUsers || [];
+    this.ircIgnoreUsers = this.ircIgnoreUsers.map(x => x.toLowerCase());
+
+    // Discord nicks to ignore (case insensitive)
+    this.discordIgnoreUsers = options.discordIgnoreUsers || [];
+    this.discordIgnoreUsers = this.discordIgnoreUsers.map(x => x.toLowerCase());
```

Comment:
maybe just compare `i.toLowerCase() === item.toLowerCase()` instead of changing the array?

Reply:
Fixed.